backend/src/config/database.py

```
@@ -67,9 +75,17 @@ def get_connection_string(self) -> str:
             return f"sqlite:///{self.get_sqlite_file_path()}"
         elif self.type == "postgresql":
-            if not all([self.postgres_host, self.postgres_user, self.postgres_password, self.postgres_database]):
-                raise ValueError("PostgreSQL connection requires host, user, password, and database")
-            return f"postgresql://{self.postgres_user}:{self.postgres_password}@{self.postgres_host}:{self.postgres_port}/{self.postgres_database}"
+            if not all([
+                self.postgres_host, self.postgres_user, 
+                self.postgres_password, self.postgres_database
+            ]):
+                raise ValueError(
+                    "PostgreSQL connection requires host, user, password, and database"
+                )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)